src/components/Home/Home.tsx

```diff
@@ -1,35 +1,66 @@
 import React, { useState, useEffect } from 'react'
+import { fetchGetResponse } from '../../utils/apiCalls'
+import { cleanMarketsData, cleanDetailsData } from '../../utils/utils'
 
 interface ApiMarkets {
   markets: {
     id: number,
+    distanceFromZip: number,
     marketName: string
-    //need logic to split the distance from the name to display the name
   }[],
   marketDetails: {
-    googlink: string,
-    address: string,
-    schedule: string,
-    products: string
-    // need logic for splitting products string into an array
+    street: string,
```

Comment:
Great job updating this interface into more specific, understandable chunks of information. This should make things more nimble when manipulating the fetched data, and therefore easier to display.